developer/src/kmc-ldml/src/compiler/ldml-compiler-messages.ts

```diff
@@ -258,4 +258,9 @@ export class LdmlCompilerMessages {
     \`<transform from="($[fromSet])" to="$[1:toSet]"/>\`.
   `);
 
+  static ERROR_UnparseableTransformTo   = SevErrorTransform | 0x06;
+  static Error_UnparseableTransformTo   = (o: { to: string, message: string }) =>
+  m(this.ERROR_UnparseableTransformTo,    `Invalid transform to="${def(o.to)}": "${def(o.message)}"`);
```

Comment:
```suggestion
  static ERROR_UnparseableTransformTo = SevErrorTransform | 0x06;
  static Error_UnparseableTransformTo = (o: {to: string, message: string}) => m(
    this.ERROR_UnparseableTransformTo,
    `Invalid transform to="${def(o.to)}": "${def(o.message)}"`,
  );
```

We are working with this formatting now across all our message files, slowly moving messages over. (Final comma allows for addition of detailed explanation parameter in Markdown format that is published to help.keyman.com)